### PR TITLE
ref: Cleanup auth registration page styling

### PR DESCRIPTION
**Pull request by EvanPurkhiser**
image
↓
image

### CODE REVIEW

File: sentry_auth_onelogin/templates/sentry_auth_onelogin/select-idp.html [hunk 1]
```diff
@@ -1,72 +1,56 @@
...
+      class="textinput textInput form-control"
+      id="id_or_metadata_url"
+      name="id_or_metadata_url"
+      type="text"
+      value="{{ id_or_metadata_url }}" />
+  </div>
 
+  <fieldset class="form-actions">
+    <button type="submit" class="btn btn-primary" name="action_save">{% trans "Continue" %}</button>
+    <a class="pull-right" style="margin-top: 7px" href="https://docs.sentry.io/learn/sso/#onelogin">Need help finding your Identity Provider?</a>
```
Comment (mattrobenolt) [inline sentry_auth_onelogin/templates/sentry_auth_onelogin/select-idp.html]: I think the wording here is wrong. I think it should be something like, "Need help finding your ID or metadata URL?"

File: sentry_auth_onelogin/templates/sentry_auth_onelogin/select-idp.html [hunk 2]
```diff
@@ -1,72 +1,56 @@
...
+  {% endif %}
+  {% if error_value %}
+    <div class="alert alert-block alert-error">
+      {% trans "The value is invalid, ID number or url expected." %}
+    </div>
+  {% endif %}
+
+  <div class="form-group">
+    <label for="id_or_metadata_url" label="control-label requiredField">
+      Identity Provider / Metadata URL
```
Comment (mattrobenolt) [inline sentry_auth_onelogin/templates/sentry_auth_onelogin/select-idp.html]: Again, I think the wording here regressed. OneLogin is the identity provider. You can plug in a numeric connector id or the full metadata url.

File: sentry_auth_onelogin/templates/sentry_auth_onelogin/select-idp.html (hunk 1 above)
Comment (EvanPurkhiser) [reply sentry_auth_onelogin/templates/sentry_auth_onelogin/select-idp.html]: Thanks, this is the feedback I was looking for.

Comment (EvanPurkhiser) [reply sentry_auth_onelogin/templates/sentry_auth_onelogin/select-idp.html]: 9be35876c64a0227f839ac066fe46386c800a269

**Pull request merged**